db/seeds.rb

```diff
@@ -7,3 +7,10 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+# Create the default user
+family = Family.create(name: "The Smiths")
```

Comment:
```suggestion
family = Family.create_or_find_by!(name: "The Smiths")
```
Maybe: https://apidock.com/rails/ActiveRecord/Relation/create_or_find_by